src/JustSaying/Messaging/MessageHandling/ExactlyOnceHandler.cs

```diff
@@ -1,22 +1,30 @@
 using System;
 using System.Threading.Tasks;
 using JustSaying.Models;
+using Microsoft.Extensions.Logging;
 
 namespace JustSaying.Messaging.MessageHandling
 {
-    public class ExactlyOnceHandler<T> : IHandlerAsync<T> where T : Message
+    internal sealed class ExactlyOnceHandler<T> : IHandlerAsync<T> where T : Message
```

Comment:
Apologies if I have missed this in a previous change but what's the reason for this no longer being public? Has the usage changed?

Reply:
I didn't see a specific use case for it being public given that it seems to be an implementation detail of elsewhere in JustSaying, so I figured I'd try and rationalise the public API surface a bit more. I can undo it if necessary, but 7's already a major change anyway so breaking changes like this are doable.